src/Main.java

```diff
@@ -0,0 +1,89 @@
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Method;
+import java.util.*;
+
+public class Main {
+    public static void main(String[] args) throws ClassNotFoundException, InstantiationException, IllegalAccessException {
+        List<MyInterface> myObjectsToScan = new ArrayList<>();
+        myObjectsToScan.add(new FirstClassImpl());
+        myObjectsToScan.add(new SecondClassImpl());
+        myObjectsToScan.add(new ThirdClassImpl());
+        myObjectsToScan.add(new FourthClassImpl());
+
+        List<Class<? extends Annotation>> customAnnotationList = new ArrayList<>();
+        customAnnotationList.add(Private.class);
+        customAnnotationList.add(Public.class);
+
+//        Task iii.1
+//        getAllClassesWithAnnotation(ScanMe.class, myObjectsToScan);
+
+//        Task iii.2
+//        getAllMethodsWithPrivateAnnotationInClassWithAnnotationScanMe(ScanMe.class, Private.class,
+//                myObjectsToScan);
+
+//        Task iii.3
+//        getAllMethodsWithoutCustomAnnotationInClassWithoutAnnotation(customAnnotationList, myObjectsToScan);
+
+//        Task iii.4
+//        getAllScannedClasses(myObjectsToScan);
+
+    }
+
+    public static void getAllClassesWithAnnotation(Class<? extends Annotation> classAnnotation, List<MyInterface> elementList) {
+        for (MyInterface element : elementList
```

Comment:
formating - form method braces in same line but on for loop in next line - correct this